packages/widgetbook/lib/src/routing/widgetbook_shell.dart

```diff
@@ -63,3 +78,144 @@ class WidgetbookShell extends StatelessWidget {
     );
   }
 }
+
+class MobileWidgetbookShell extends StatefulWidget {
+  const MobileWidgetbookShell({
+    super.key,
+    required this.child,
+  });
+
+  final Widget child;
+
+  @override
+  State<MobileWidgetbookShell> createState() => MobileWidgetbookShellState();
+}
+
+class MobileWidgetbookShellState extends State<MobileWidgetbookShell> {
+  int _pointerCount = 0;
+  bool isBottomBarVisible = true;
+
+  void handlePointerDown(PointerDownEvent event) {
+    setState(() {
+      _pointerCount++;
+    });
+
+    if (_pointerCount == 4) {
+      setState(() {
+        isBottomBarVisible = !isBottomBarVisible;
+      });
+    }
+  }
```

Comment:
Is this logic working? It seemed to me that [you won't be adding it](https://github.com/widgetbook/widgetbook/issues/805#issuecomment-1790538697).

Reply:
Yes, taken it out.